services/QuillLMS/spec/models/gengo_job_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: gengo_jobs
+#
+#  id                 :bigint           not null, primary key
+#  locale             :string           not null
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  english_text_id    :integer          not null
+#  translated_text_id :integer
+#  translation_job_id :string           not null
+#
+require 'rails_helper'
+
+RSpec.describe GengoJob, type: :model do
+  describe "self.pending_translation" do
+    subject { GengoJob.pending_translation }
+
+    let!(:untranslated) { create(:gengo_job) }
+    let!(:complete) { create(:gengo_job, translated_text_id: 2) }
```

Comment:
These two `let!` calls are repeated both here and in the next `describe` block which makes me think that there should be some `context` that wraps both `describe`s where the `let!`s are declared.

Reply:
hmm, I don't love adding the same context around two describe blocks though, that feels weird. I know it's a little reudundant but since it's a test, I don't mind so much.